src/public/modules/users/views/authentication/signin.client.view.html

```diff
@@ -82,7 +82,7 @@
           ng-keyup="$event.keyCode == 13 && vm.credentials.otp.length === 6 && vm.verifyOtp()"
         />
         <div
-          ng-if="vm.signInMsg.isMsg"
+          ng-if="!vm.showOtpDelayNotification && vm.signInMsg.isMsg"
```

Comment:
i'd think these messages take precedence over the delay notification. Maybe reverse the conditional? only show otpDelayNotification if !vm.isOtpSending || !vm.signInMsg.isMsg

Reply:
I made adjustments such that the `ng-if`s should behave like the following table:

| vm.signInMsg.isMsg | vm.showOtpDelayNotification | vm.isOtpSending |                        |
|--------------------|-----------------------------|-----------------|------------------------|
| 0                  | 0                           | 0               | Nothing                |
| 0                  | 0                           | 1               | Sending OTP...         |
| 0                  | 1                           | 0               | OTP delay notification |
| 0                  | 1                           | 1               | OTP delay notification |
| 1                  | 0                           | 0               | Sign-in messge         |
| 1                  | 0                           | 1               | Sign-in message        |
| 1                  | 1                           | 0               | Sign-in message        |
| 1                  | 1                           | 1               | Sign-in message        |

I kind of did this up in a slight rush so I will be back to verify my work once more.

Reply:
I see. I think will need more design thinking to see which message has priority. Maybe if both sending messages and showOtpDelayNotification will need a combined message. @syan-syan thoughts?

Maybe just concat `OTP might be delayed due to government email traffic` to all current messages after 45 seconds.

e.g. sending otp after message:
` Sending OTP... OTP might be delayed due to government email traffic` etc etc